### PR TITLE
docs: add guide for importing SCrypt passwords

### PR DESCRIPTION
**Pull request by LandonPattison**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

## Further comments
Added documentation for showing how to import SCrypt algorithm passwords
PR to implement scrypt algorithm importing: https://github.com/ory/kratos/pull/2689

### CODE REVIEW

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]
Thanks for your contribution! It seems that you forgot to format the documents since the `Format / format (pull_request)` CI step fails. 

Please run 
```
npm i
npm run format
```
and then commit the formatted files to fix this.

File: docs/kratos/concepts/credentials/username-email-password.mdx [hunk 1]
```diff
@@ -361,6 +361,31 @@ $pbkdf2-<algorithm>$i=<iteration>,l=<length>$<salt>$<hash>
 $pbkdf2-sha256$i=100000,l=32$1jP+5Zxpxgtee/iPxGgOz0RfE9/KJuDElP1ley4VxXc$QJxzfvdbHYBpydCbHoFg3GJEqMFULwskiuqiJctoYpI
 ```
 
+### SCrypt
+
+Ory Kratos doesn't hash passwords by SCrypt but can compare stored SCrypt hash and migrate to configured hasher
```
Comment (kevgo) [inline docs/kratos/concepts/credentials/username-email-password.mdx:366]:
```suggestion
Ory Kratos doesn't hash passwords by SCrypt but can compare stored SCrypt hashes and migrate to configured hasher
```

File: docs/kratos/concepts/credentials/username-email-password.mdx [hunk 2]
```diff
@@ -361,6 +361,31 @@ $pbkdf2-<algorithm>$i=<iteration>,l=<length>$<salt>$<hash>
...
+
+```
+$scrypt$N=<cost>,r=<block>,p=<parrelization>$<salt>$<hash>
+```
+
+#### Parameters
+
+- `cost`(`number`): CPU/memory cost (has to be power of 2 and >1)
+- `block`(`number`): block size parameter (must satisfy r * p < 2³⁰)
+- `parrelization`(`number`): parallelization parameter (must satisfy r * p < 2³⁰)
```
Comment (kevgo) [inline docs/kratos/concepts/credentials/username-email-password.mdx:379]:
```suggestion
- `parallelization`(`number`): parallelization parameter (must satisfy r * p < 2³⁰)
```

**Review by kevgo — COMMENTED** [review]
Thanks for adding this! A few syntax improvements below.

File: docs/kratos/concepts/credentials/username-email-password.mdx (hunk 1 above)
Comment (LandonPattison) [reply docs/kratos/concepts/credentials/username-email-password.mdx]: Fixed :)

File: docs/kratos/concepts/credentials/username-email-password.mdx (hunk 2 above)
Comment (LandonPattison) [reply docs/kratos/concepts/credentials/username-email-password.mdx]: fixed :)

**Comment by tomekpapiernik** [comment]
Looks okay now, good to merge once the related code is merged too. @aeneasr

**Review by tomekpapiernik — APPROVED** [review]

**Pull request merged**